deno/payloads/common.ts

```diff
@@ -59,6 +59,7 @@ export const PermissionFlagsBits = {
 	 * @unstable This permission flag is currently not documented by Discord but has a known value which we will try to keep up to date.
 	 */
 	UseExternalSounds: 1n << 45n,
+	SendVoiceMessages: 1n << 46n,
```

Comment:
Why is this not documented?

Reply:
What do you mean?

Reply:
Ah, don't worry. I just thought these would be documented, but none of them are.

Reply:
Should they be documented at some point?

Reply:
Maybe a simple description could be added for them in the future, but that's not a concern for this pull request (out of scope). This is fine as-is.